tools/fab/frontend-install.py

```diff
@@ -61,18 +61,13 @@ def banner(message):
 	print('#######################################')
 
 
-def copy(source, dest):
```

Comment:
What prompted changes to `mount` and `copy`? Did adding probepal require modifying their behavior?

Reply:
Just found while debugging.

`copy` was dead code (not called anywhere) before I got there.  `mount` suppressed errors.